DESCRIPTION

```diff
@@ -10,14 +10,18 @@ Authors@R: c(person("Bioconductor Package Maintainer", role = c("aut", "cre"),
 	      person("Valerie", "Obenchain", role = "aut"),
 	      person("Michael", "Love", role = "aut"),
 	      person("Lori", "Shepherd", role = "aut"),
-	      person("Martin", "Morgan", role = "aut"))
+	      person("Martin", "Morgan", role = "aut"),
+              person("Sonali", "Kumari", role = "ctb",
+              comment = "Converted GinomicFles vignette from GenomicFiles.Rnw to RMarkdown / HTML."),
+        person("Bioconductor Package Maintainer",
+        email = "maintainer@bioconductor.org", role = "cre"))
 biocViews: Genetics, Infrastructure, DataImport, Sequencing, Coverage
 Depends: R (>= 3.1.0), methods, BiocGenerics (>= 0.11.2), MatrixGenerics,
         GenomicRanges (>= 1.31.16), SummarizedExperiment,
         BiocParallel (>= 1.1.0), Rsamtools (>= 1.17.29), rtracklayer (>= 1.25.3)
 Imports: GenomicAlignments (>= 1.7.7), IRanges, S4Vectors (>= 0.9.25),
         VariantAnnotation (>= 1.27.9), GenomeInfoDb
-Suggests: BiocStyle, RUnit, genefilter, deepSNV, snpStats,
+Suggests: BiocStyle, RUnit, genefilter, deepSNV, snpStats,BiocStyle,knitr
```

Comment:
BiocStyle already exists in Suggests. Please remove the additional BiocStyle. Leave `knitr` but add a space before the comma.

Reply:
I have made all the changes